### DISCUSSION TITLE
Can't install on VSC 1.70.0

### DISCUSSION THREAD

**Discussion by TonyThorne** [antfu/retypewriter#6] [Q&A] [answered]
When trying to add the extension via VSC the following message is displayed. 

Can't install 'antu.retypewriter' extension because it is not compatible with VSC 1.70.0

Version: 1.70.0 (user setup)
Commit: da76f93349a72022ca4670c1b84860304616aaa2
Date: 2022-08-04T04:38:16.462Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.19044

![reType](https://user-images.githubusercontent.com/6304471/194552473-ce854fc6-01a4-448a-a3cb-794c2c5cb6ee.PNG)

**Comment by antfu** [comment, marked as answer]
It requires 1.71.0 https://github.com/antfu/retypewriter/blob/dcf95a34a1951269cd9e258c46e85b2dd06ad14b/packages/vscode/package.json#L26

Any blockers for the upgrade?

**Reply by TonyThorne** [reply]
Thanks. My work machine is a managed device, so I could not manually update it. I have now tried it on my home PC and it works perfectly. Apologies, I didn't realise the VSC version would be in the package.json. (Every day is a school day!). Excited to start using it. Thanks for creating it.
